src/components/cc-tcp-redirection-form/cc-tcp-redirection-form.js

```diff
@@ -62,32 +63,33 @@ export class CcTcpRedirectionForm extends LitElement {
           <div class="description">${i18n('cc-tcp-redirection-form.description')}</div>
         ` : ''}
 
-        ${state === 'loading' ? html`
-          ${SKELETON_REDIRECTIONS.map((redirection) => html`
-            <cc-tcp-redirection .redirection=${redirection}></cc-tcp-redirection>
+        ${this.state.type === 'error' ? html`
+            <cc-notice intent="warning" message="${i18n('cc-tcp-redirection-form.error')}"></cc-notice>
+        ` : ''}
+
+        ${this.state.type === 'loading' ? html`
```

Comment:
nitpick: You could merge the 'loading' and 'loaded' HTML logic with a unique list of redirections. Where this list of redirection is initialized before, according to the loading/loaded state.